cmd/snap-seccomp/main.go

```diff
@@ -918,7 +918,7 @@ func compile(content []byte, out string) error {
 		if err != nil {
 			return fmt.Errorf("cannot create allow seccomp filter: %s", err)
 		}
-		secFilterDeny, err = seccomp.NewFilter(complainAct)
+		secFilterDeny, err = seccomp.NewFilter(seccomp.ActAllow)
```

Comment:
I think I'd add a comment explaining that the default action is allow and similarly to the default filter we will deny selected actions, but using complain/log action.

Also, I'd add a simialr comment in the default case below.

Reply:
I've added comments in both branches.